index.js

```diff
@@ -14,6 +14,21 @@
 \*───────────────────────────────────────────────────────────────────────────*/
 'use strict';
 var fs = require('fs');
+var path = require('path');
+fs.mkdirRecursive = function(dirPath, mode) {
+	try {
+		fs.mkdirSync(dirPath, mode);
+	}
+	catch (error) {
+		if (error.code === 'EEXIST' || error.errno === 34) {
+			fs.mkdirRecursive(path.dirname(dirPath), mode);
+			fs.mkdirRecursive(dirPath, mode);
+		}
+		else {
+            throw error;
```

Comment:
I don't think we want to throw any errors from plugins. I would suggest to just log the error (`console.error(error)`) and terminate the flow by calling `callback()`